File: src/main/java/com/uiptv/service/remotesync/RemoteSyncProgressStep.java
package com.uiptv.service.remotesync;

public enum RemoteSyncProgressStep {
    CONNECTING,
    WAITING_FOR_APPROVAL,
    CREATING_SNAPSHOT,
    UPLOADING,
    PREPARING_DOWNLOAD,
    DOWNLOADING,
    APPLYING_SYNC,
    COMPLETING_REMOTE,
    FINISHED
}

File: server/src/main/kotlin/com/uiptv/service/remotesync/RemoteSyncProgressStep.kt
package com.uiptv.service.remotesync

enum class RemoteSyncProgressStep {
    CONNECTING,
    WAITING_FOR_APPROVAL,
    CREATING_SNAPSHOT,
    UPLOADING,
    PREPARING_DOWNLOAD,
    DOWNLOADING,
    APPLYING_SYNC,
    COMPLETING_REMOTE,
    FINISHED
}
